### ISSUE TITLE
SyntaxError - Unexpected token function

### ISSUE THREAD

**Issue by quotschmacher** [bryanbcook/azdevops-testplan-extension#6]
**Description**
I get Syntax Error at each run

Right before this task I have another task:
```yaml
steps:
- script: |
   echo $(System.DefaultWorkingDirectory)
   
   ls $(System.DefaultWorkingDirectory)/testcafe
```

giving me the following output:
```bash
/home/vsts/work/1/s
Readme.md
configurations
errorMessages.txt
models
node_modules
package-lock.json
package.json
report.xml
tests
```

So the report.xml file is present with the following content:
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<testsuite name="TestCafe Tests: Firefox 122.0 / Ubuntu 22.04" tests="2" failures="0" skipped="0" errors="0" time="218.781" timestamp="Fri, 09 Feb 2024 13:10:34 GMT" >
  <testcase classname="Neuland Tests" file="/home/vsts/work/1/s/testcafe/tests/testNld.js" name="LogIn" time="18.405">
  </testcase>
  <testcase classname="Neuland Tests" file="/home/vsts/work/1/s/testcafe/tests/testNld.js" name="Auftrag erfassen" time="197.758">
  </testcase>
</testsuite>
```

```yaml
steps:
- task: bcook.azdevops-testplan-extension.publishtestplanresults.PublishTestPlanResults@0
  displayName: 'Publish Test Results'
  inputs:
    testPlan: 'UI Test mit Testcafe'
    testResultFormat: xUnit
    testResultDirectory: '$(System.DefaultWorkingDirectory)/testcafe/'
    testResultFiles: report.xml
```

giving me the following results:

```bash
Starting: Publish Test Results
==============================================================================
Task         : Publish Test Plan Results
Description  : Publishes cross-platform test results (xUnit, JUnit) to an Azure DevOps Test Plan
Version      : 0.1.4
Author       : Bryan Cook
Help         : 
==============================================================================
/home/vsts/work/_tasks/PublishTestPlanResults_.../0.1.4/index.js:31
async function run() {
      ^^^^^^^^

SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:549:28)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
##[error]Exit code 1 returned from process: file name '/home/vsts/agents/3.232.3/externals/node/bin/node', arguments '"/home/vsts/work/_tasks/PublishTestPlanResults_.../0.1.4/index.js"'.
Finishing: Publish Test Results
```

With the following environment:
- Pool: [Azure Pipelines]
- Image: ubuntu-22.04
- Agent: Hosted Agent

Did I configure something wrong? Can I support you with some more information?

**Comment by bryanbcook** [comment]
Thanks for reporting this. I will investigate.

**Comment by bryanbcook** [comment]
@quotschmacher - I was able to reproduce your issue in a Cloud-hosted agents which default to a lower incompatible version of Node.  (I was running my agent locally using Node 18)

This should unblock you.

Your feedback is very valuable. Look forward to hearing about your progress.
